recipe/meta.yaml

```diff
@@ -10,21 +10,23 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - windows-bat.patch  # [win]
```

Comment:
Please move the comment about the need of this patch above this line.

Reply:
**You got it.** This comment *was* originally situated above that exact line, for orthogonality with the python-graphviz feedstock. However, this caused conda-forge-linter to [break with non-human-readable non-errors](https://github.com/conda-forge/pydot-feedstock/pull/2#issuecomment-412254439). In the interest of *just shutting up* the linter, I then shifted that commentary out into the patch.

*Annnyway.* As you suggest, who cares about the linter? Recipe maintainability takes precedence. The linter's fundamentally broken as currently defined, anyway. <sup>*Christ, someone fix that linter.*</sup>

I have the sudden feeling I talk too much.